service/src/main/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleService.java

```diff
@@ -55,8 +55,8 @@ public boolean isNewCaseStateModelEnabled() {
         return ldClient.boolVariation("new-case-state-model", createLDUser(), false);
     }
 
-    public boolean isMigrateToManageOrgWarningPageEnabled(String localAuthorityName) {
-        return ldClient.boolVariation("migrate-to-manage-org-warning-page",
+    public boolean isBlockCasesForLocalAuthoritiesNotOnboardedEnabled(String localAuthorityName) {
```

Comment:
would not it be better to have opposite toggle AllowCaseCreationForNonOnboardedUsers? It should be exception that we allow it, so insteda specyfiying all our LAs in toggle we should specify only exceptions to rule